gstd/src/msg/async.rs

```diff
@@ -90,7 +99,7 @@ impl FusedFuture for MessageFuture {
     }
 }
 
-/// Send a message and wait for reply.
+/// Send a message and wait for reply (as encoded structure).
```

Comment:
I would rephrase a bit - specify that wait for reply and returns result as a structure versus same thing with raw reply message payload